src/model/reddit.ts

```diff
@@ -819,24 +817,35 @@ export async function getSearchArticle(data: IAPIData): Promise<IGenericResult<I
 export async function loginUser(data: IAPIData): Promise<any> {
     return new Promise<any>(async (resolve, reject) => {
         if (secureBaseHost && data.username && data.password) {
-            const rest: rm.RestClient = new rm.RestClient('login.user', secureBaseHost, undefined, { socketTimeout: data.timeout });
-            let url: string = "api/login/" + encodeURIComponent(data.username) + "?api_type=json&user=" + encodeURIComponent(data.username) + "&passwd=" + encodeURIComponent(data.password);
-            rest.create<any>(url, {
-                additionalHeaders: {
-                    "user-agent": "Reddit-Viewer / 2.0.0"
+            var request = require('request');
+            var options = {
```

Comment:
Bad practice. **VAR** is global scope. Use **LET** instead.

Reply:
Yeah, I should have worn you that I don't have a lot of experience in JavaScript :sweat_smile: 
I'll fix that next time